### ISSUE TITLE
Error when review deactivated for author

### ISSUE THREAD

**Issue by eddoff** [pkp/pkp-lib#5109]
For OJS 3.1.2-1

When the "Review" is deactivated for "Author" in User & Roles > Roles, then there is an error when trying to reach the submission as author.
![image](https://user-images.githubusercontent.com/41610473/65882271-07ae3380-e395-11e9-945b-9465eaaf50d7.png)

/eddoff

**Comment by asmecher** [comment]
@eddoff, while that's definitely not a graceful way to present disallowed content, removing the "review" checkmark is intended to prevent the author from accessing the review stage of the submission, which is what you're seeing. Can you describe what your workflow goal is? Do you want e.g. to hide the "Review" tab entirely from authors?

**Comment by eddoff** [comment]
Yes, I did expect to hide the review tab for the author. At least I did not expect error messages such as that one. Better a error message such as: "You are not allowed to show this information"
